packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -264,6 +264,10 @@ When('I click the {string} button', (name: string) => {
   }).click();
 });
 
+Then('I press the {string} key', (key: string) => {
+  cy.get('body').type(key);
+});
+
```

Comment:
Adding for the ability to press the `esc` key to dismiss the file picker which pops up when navigating to the Upload Action view